### ISSUE TITLE
Epitrace age in bulk vs single-cell ATAC data

### ISSUE THREAD

**Issue by HaikuoLi** [MagpiePKU/EpiTrace#13]
Hello!

Thank you mentioning to us again that the computed EpiTrace age is negatively correlated with mitotic age for bulk datasets and positively correlated for single-cell datasets. But I still feel unclear of the reason of this, and how to interpret my result.

To my understanding, functions in EpiTrace do not specify whether we are working on a bulk or single-cell dataset. That means, for example, if I am using `EpiTraceAge_Convergence`, I can always get some kind of results of EpiTrace age regardless of whether I am processing bulk or snATAC-seq data, but should I manually reverse the results (for example in your paper you did `1-rank`) or it has already been reversed? Do either `EpiTraceAge_Convergence` or `RunEpiTraceAge` presume that it is processing a bulk or sn data?

Thank you!

**Comment by MagpiePKU** [comment]
Yes you are correct that the algorithm does NOT assume data type. You should manually reverse the results  for bulk.
